pkgs/applications/graphics/meme/default.nix

```diff
@@ -1,19 +1,20 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub, runCommand }:
 
-buildGoPackage rec {
+buildGoModule rec {
```

Comment:
Why switch to modules when upstream has a `/vendor`?

Reply:
https://github.com/golang/go/wiki/Modules#go-modules

In general, modules are the supported way forward. vendor directories are mostly used to provide backwards compatibility, as well as save time downloading code (i.e. go install -mod=vendor is default if there is a vendor directory). As long as a go.mod and go.sum file exist in the repo, we should flip over to using it.

Reply:
Also updated the PR motivation.

Reply:
See https://github.com/NixOS/nixpkgs/issues/84826 for some discussion about this.

Reply:
In the long term we should also introduce a `go2nix` for `buildGoModule`.